code/src/hooks/use-dropdown/useDropdown.ts

```diff
@@ -0,0 +1,21 @@
+import { useState } from "react";
+
+const useDropdown = () => {
+  const [isDropdownOpened, setIsDropdownOpened] = useState(false);
+
+  const handleCloseDropdown = () => {
+    setIsDropdownOpened(false);
+  };
+
+  const handleOpenDropdown = () => {
+    setIsDropdownOpened(true);
+  };
+
```

Comment:
Can we add toggle for extensibility, just in case? Because we do have toggle for modal and drawer.

Reply:
added `toggleDropdown`